GameDataParser/Parsers/MapEntityParser.cs

```diff
@@ -129,6 +131,30 @@ protected override List<MapEntityMetadata> Parse()
                         CoordB coord = CoordB.Parse(coordMatch.Value, ", ");
                         metadata.Objects.Add(new MapObject(coord, int.Parse(mapObjects[modelName])));
                     }
+                    else if (NpcEntityMatch.Success)
+                    {
+                        // Skip some items we don't handle yet
+                        if (NpcEntityMatch.Groups[0].Value.Contains("WayPoint"))
+                        {
+                            continue;
+                        }
+                        else if (NpcEntityMatch.Groups[0].Value.Contains("PatrolData"))
+                        {
+                            continue;
+                        }
+
+                        int npcId = int.Parse(NpcEntityMatch.Groups[1].Value.Split("_")[0]);
```

Comment:
Can you move this line below 152? That way we don't parse this unless we actually need it. Also, the same can be said for the `playerRotation`

Reply:
Fixed